receiver/prometheusreceiver/DESIGN.md

```diff
@@ -688,4 +688,4 @@ will produce an OpenTelemetry Summary with Snapshot set to `nil`.
 ### Others
 
 For any other Prometheus metrics types, they will be transformed into the
-OpenTelemetry [Gauge](#gauge) type.
+OpenTelemetry [Gauge](https://prometheus.io/docs/concepts/metric_types/#guage) type.
```

Comment:
I believe this link is incorrect. It should be pointed to OTel spec, not prometheus